tgui/packages/tgui/interfaces/PreferencesMenu/preferences/features/game_preferences/fov_darkness.tsx

```diff
@@ -0,0 +1,7 @@
+import { Feature, FeatureNumberInput } from "../base";
+
+export const fov_darkness: Feature<number> = {
+  name: "FoV Darkness",
```

Comment:
Please write out the full name and also a description of what this is, I didn't even know what FoV was referring to when it was being discussed. Like, yes, I know it's field of view, but not how that related to the game.